### PR TITLE
Doc changes Azure ASE

### PR DESCRIPTION
**Pull request by ankurch627**
## Purpose of this pull request

This pull request... 

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/microsoft-azure/azure-app-service-environment.md [hunk 1]
```diff
@@ -38,3 +84,63 @@ In this section, you will configure a pipeline for shipping diagnostic logs from
    * Select `App Service Environment Platform Logs`.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-app-service-environment.md:84]: add screenshot for diagnostic settings

File: docs/integrations/microsoft-azure/azure-app-service-environment.md [hunk 2]
```diff
@@ -38,3 +84,63 @@ In this section, you will configure a pipeline for shipping diagnostic logs from
...
+
+## Viewing Azure App Service Environment dashboards
+
+import ViewDashboards from '../../reuse/apps/view-dashboards.md';
+
+<ViewDashboards/>
+
+### Overview
+
+The **Azure AppService - Overview** dashboard provides comprehensive information of all the service health incidents or resource health events associated with Azure AppService in your azure account.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-app-service-environment.md:107]: dashboard names should be starting with `Azure App Service Environment - `

File: docs/integrations/microsoft-azure/azure-app-service-environment.md [hunk 3]
```diff
@@ -38,3 +84,63 @@ In this section, you will configure a pipeline for shipping diagnostic logs from
...
+
+Use this dashboard to:
+* View recent resource and service health incidents.
+* View distribution of service and resource health by incident type.
+
+<img src={useBaseUrl('https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Azure-AppServiceEnvironment/Azure-App-Service-Plan-Overview.png')} alt="Azure AppServiceEnvironment Overview dashboard" style={{border: '1px solid gray'}} width="800" />
+
+### Operations
+
+The **Azure AppService - Operations** dashboard offers comprehensive insights into the scaling, upgrade events for your Azure AppServiceEnvironment.
```
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-app-service-environment.md:117]: AppServiceEnvironment break it down to separate words

File: docs/integrations/microsoft-azure/azure-app-service-environment.md (hunk 2 above)
Comment (himanshu219) [inline docs/integrations/microsoft-azure/azure-app-service-environment.md:107]: associated with Azure AppService -> associated with Azure App Service Environments

**Review by himanshu219 — CHANGES_REQUESTED** [review]
minor changes

File: docs/integrations/microsoft-azure/azure-app-service-environment.md (hunk 2 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-app-service-environment.md]: Done

File: docs/integrations/microsoft-azure/azure-app-service-environment.md (hunk 3 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-app-service-environment.md]: Done

File: docs/integrations/microsoft-azure/azure-app-service-environment.md (hunk 2 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-app-service-environment.md]: Done

**Comment by jpipkin1** [comment]
@ankurch627, I converted the PR to a draft because you have "Draft" in the title. When the PR is ready for review, click the **Ready for review** button.

**Comment by ankurch627** [comment]
@jpipkin1 unable to find the Ready for review button. Can you please help? The PR is ready for review

File: docs/integrations/microsoft-azure/azure-app-service-environment.md [hunk 4]
```diff
@@ -8,33 +8,140 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+#### Diagnostic logs
+
+In this section, you will configure a pipeline for shipping diagnostic logs from the Azure Monitor to an Event Hub.
 
 1. To set up the Azure Event Hubs source in Sumo Logic, refer to [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
-2. To create the Diagnostic settings in Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings?tabs=portal#create-diagnostic-settings). Perform below steps for each Azure App Service Environment that you want to monitor.
-   * Choose `Stream to an event hub` as the destination.
-   * Select `App Service Environment Platform Logs`.
-   * Use the Event hub namespace and Event hub name configured in previous step in destination details section. You can use the default policy `RootManageSharedAccessKey` as the policy name.
+2. To create the **Diagnostic setting** in the Azure portal, refer to the [Azure documentation](https://learn.microsoft.com/en-gb/azure/data-factory/monitor-configure-diagnostics). Perform the below steps for each Azure WebApps that you want to monitor.
```
Comment (JV0812) [inline docs/integrations/microsoft-azure/azure-app-service-environment.md:81]: Is it "Perform the below steps for each Azure WebApps that you want to monitor" or "Perform the below steps for each Azure App Service Environment that you want to monitor"

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor comments.

File: docs/integrations/microsoft-azure/azure-app-service-environment.md (hunk 1 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-app-service-environment.md]: Added

File: docs/integrations/microsoft-azure/azure-app-service-environment.md (hunk 4 above)
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-app-service-environment.md:81]: Fixed

File: docs/integrations/microsoft-azure/azure-app-service-environment.md [hunk 5]
```diff
@@ -8,33 +8,140 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
...
+Azure service sends monitoring data to Azure Monitor, which can then [stream data to Eventhub](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/stream-monitoring-data-event-hubs). Sumo Logic supports logs collection from [Azure Monitor](https://docs.microsoft.com/en-us/azure/monitoring-and-diagnostics/monitoring-get-started) using our [Azure Event Hubs source](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
 
 You must explicitly enable diagnostic settings for each Azure App Service Environment you want to monitor. You can forward logs to the same event hub provided they satisfy the limitations and permissions as described [here](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings?tabs=portal#destination-limitations).
 
 When you configure the event hubs source or HTTP source, plan your source category to ease the querying process. A hierarchical approach allows you to make use of wildcards. For example: `Azure/AppServiceEnvironment/Logs`, `Azure/AppServiceEnvironment/Metrics`.
```
[the remark this answers is not in the document]
Comment (ankurch627) [reply docs/integrations/microsoft-azure/azure-app-service-environment.md:32]: Done

**Review by himanshu219 — APPROVED** [review]

**Pull request merged**